### DISCUSSION TITLE
I can't seem to find the Project/repo readme generator that this repository has used, I've used it previously aswell, but I cant seem to find it anymore, can anyone please help out?

### DISCUSSION THREAD

**Discussion by DepthDweller21** [rahuldkjain/github-profile-readme-generator#800] [Q&A]
there used to be a repo readme generator but I cant find it anymore but it seems like this repo has used the exact same one aswell, if someone can help me find it then that would be great

**Comment by rahuldkjain** [comment]
sorry didn't get you. Please explain it again.
